worker/worker.py

```diff
@@ -232,6 +233,8 @@ def main():
   success = True
   global ALIVE
   while ALIVE:
+    if path.isfile("/tmp/fish.exit"):
```

Comment:
```
import tempfile

    if path.isfile(path.join(tempfile.gettempdir(), 'fish.exit')):
```

Reply:
or maybe in testing folder instead? like custom_make.txt.

Reply:
@noobpwnftw I approve your proposition. A user running hundreds of workers is able to code a script to write/delete the file. On unix this should work
```bash
touch dir{00..99}/worker/fish.exit
rm dir{00..99}/worker/fish.exit
```

Reply:
@tomtor file in the worker dir, this should work:
```
   if path.isfile(path.join(os.path.dirname(os.path.realpath(__file__)), 'fish.exit')):
```

Reply:
I pushed your code suggestion, thanks

Reply:
@tomtor I edited the suggestion to set `worker_dir` at the script start because after the first 250 game batch `os.path.dirname(os.path.realpath(__file__)=/<some_path>/worker/testing`